click_odoo_contrib/update.py

```diff
@@ -113,10 +217,20 @@ def OdooEnvironmentWithUpdate(database, ctx, **kwargs):
 @click.option(
     "--if-exists", is_flag=True, help="Don't report error if database doesn't exist"
 )
-def main(env, i18n_overwrite, update_all, if_exists):
+@click.option(
+    "--watcher-max-seconds",
+    default=0,
+    type=float,
+    help="Max DB lock seconds allowed before aborting the update process",
+)
+def main(env, i18n_overwrite, update_all, if_exists, watcher_max_seconds):
     """ Update an Odoo database (odoo -u), automatically detecting
     addons to update based on a hash of their file content, compared
     to the hashes stored in the database.
+
+    It allows updating in parallel while another Odoo instance is still
+    running against the same database, by using a watcher that aborts the
+    update in case a DB lock happens.
```

Comment:
Can you update the relevant section in README with the new help text and the new options?